doc/modules/clustering.rst

```diff
@@ -1569,3 +1569,77 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _davies–bouldin_index:
+
+Davies–Bouldin Index
+----------------------
+
+If the ground truth labels are not known, the Davies–Bouldin index
+(:func:`sklearn.metrics.davies_bouldin_index`) can be used to evaluate the
+model, where a lower Davies–Bouldin Index relates to a model with better
+separation between clusters.
+
+For :math:`k` clusters, the Davies–Bouldin index :math:`DB` is given as the
+ratio of within cluster-mean distance to the between means distance.
+
+.. math::
+  DB(k) = \frac{1}{k} \sum_{i=1}^k \max_{i \neq j} D_{ij}
+
+Where :math:`D_{ij}` is the ratio between the within distances in clusters
+:math:`i` and :math:`j` and the distance between the means of cluster
+:math:`i` and :math:`j`.
+
+.. math::
+  D_{ij} = \frac{\bar{d_i}+\bar{d_j}}{d_ij}
+
+:math:`\bar{d_i}` is the average distance between each point in cluster
+:math:`i` and the centroid of cluster :math:`i`.
```

Comment:
I think, "known as its diameter." Then you may add a comment that dbar_j is similarly the diameter for cluster j, or you can just leave it out, because that's clear from the notation.